styles.css

```
@@ -0,0 +1,446 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300;400;500;700&family=Roboto:wght@300;400;500;700&display=swap');
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --purple-color: #5722CD;
+    --blue-color: #4845E4;
+    --pink-color: #ED4385;
+    --black-color: #0C0C0C;
+    --lightgrey-color: #F3F6F9; /*(voor de achtergrond van de projecten)*/
+    --bg-words-color: #F2F0FF; /*(voor de grote zwevende woorden, zoals "hello" en "services") Uitzonderingen:*/
+    --gray-text-color: #7A7A7A; /*(voor de service-tegel tekst)*/
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    width: 100vw;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 300;
+    font-size: 20px;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+/*font-family: 'Merriweather', serif;*/
+/*font-family: 'Roboto', sans-serif;*/
+h1, h2, h3, #intro {
+    font-family: 'Merriweather', serif;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+.outer-container {
+    display: flex;
+    justify-content: center;
+    position: relative;
+}
+
+.inner-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    max-width: 1400px;
+    margin: 40px 40px;
```

Dit kun je verkorten:
```suggestion
    margin: 40px;
```

---

klopt helemaal